### PR TITLE
New function template to allow users creating their own basis functions (bf) from a data vector

### LINKED ISSUE #236: Generic response function
### Feature Description

This links to previous issues #148 and #231.

Wrapper function that reads a user-defined data vector (which describes an empirical RF) and user-defined sampling rate for that data vector, and generates an RF at PsPM-defined sample rate.

### PR DESCRIPTION
**Pull request by dominikbach**
Fixes #236 #231  .

Changes proposed in this pull request:
- see title

### CODE REVIEW

File: src/pspm_bf_data.m [hunk 1]
```diff
@@ -0,0 +1,42 @@
+function [bf, x] = pspm_bf_data(td)
```
Comment (teddychao) [inline src/pspm_bf_data.m:1]: Can we add some description about the input and output variables?

Comment (teddychao) [reply src/pspm_bf_data.m:1]: I will add some explanation in the manual and release notes

**Review by teddychao — APPROVED** [review]

**Comment by teddychao** [comment]
@dominikbach Could I know if I need to write a test script for this function? Thanks

**Comment by dominikbach** [comment]
@teddychao That would be great!

**Pull request merged**